apps/eth-wrapper/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "eth-wrapper",
+  "homepage": "./",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@gnosis.pm/safe-apps-ethers-provider": "^0.1.0",
+    "@gnosis.pm/safe-apps-react-sdk": "2.0.0",
+    "@gnosis.pm/safe-react-components": "^0.5.0",
+    "@material-ui/core": "^4.11.0",
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.2.2",
+    "@testing-library/user-event": "^12.1.3",
+    "@types/jest": "^26.0.10",
+    "@types/node": "^14.6.2",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@types/styled-components": "^5.1.2",
+    "ethers": "^5.0.29",
+    "react": "^17.0.1",
+    "react-app-rewired": "^2.1.6",
```

Comment:
This should be a dev dependency and actually it is not required in this setup as this project is part of a monorepo. See the other projects for how it should look like.

Reply:
dependency removed. I modified scripts to be like the other apps.